rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/executor/BakePoller.groovy

```diff
@@ -120,6 +125,11 @@ class BakePoller implements ApplicationListener<ContextRefreshedEvent> {
                             if (!cancellationSucceeded) {
                               bakeStore.removeFromIncompletes(roscoInstanceId, statusId)
                             }
+
+                            // This will have the most up-to-date timestamp.
```

Comment:
oh, so this is really just an orphan due to a timeout and you are presuming that the managing rosco instance is negligent (e.g. died). I was assuming that we were the ones ophaning the task. A comment would be useful here.